app/src/main/res/values-cs/strings.xml

```diff
@@ -53,6 +53,10 @@
   <!-- //////////////// Startup //////////////////// -->
 
   <string name="startup_wait">Nahrávám…</string>
+  <string name="startup_wait_tor">Spouštím Tor…</string>
+  <string name="startup_start_button">Restartovat</string>
+  <string name="startup_settings_button">Nastavení</string>
+  <string name="startup_error_tor">Nepodařilo se spustit Tor. Zkontrolujte, že vaše připojení je funkční a zkuste to znovu.\n\nDetail chyby: %1$s</string>
```

Comment:
<string name="startup_error_tor">Nepodařilo se spustit Tor. Zkontrolujte, že je vaše připojení funkční a zkuste to znovu.\n\nDetail chyby: %1$s</string>